### ISSUE TITLE
When core tilezen repos tagged for release, also release via CI to PyPi

### ISSUE THREAD

**Issue by nvkelso** [tilezen/tilezen-tasks#2]
Is this even possible? ;)

- vector-datasoruce
- tileserver
- tilequeue

**Comment by rmarianski** [comment]
I'd rather it be done manually to have complete control over it. The releases are manual anyway, in that we write a changelog. Uploading it just running a single command.

If we did want to automate pushing to pypi, we'd have to put someone's credentials in there anyway, or create a separate bot account.

**Comment by nvkelso** [comment]
I've added a (manual) step for **PyPi** to the https://github.com/mapzen/tile-tasks/wiki/Deployment-Checklist used to construct prod go-live tickets like https://github.com/mapzen/tile-tasks/issues/256.

**Comment by nvkelso** [comment]
> The releases are manual anyway, in that we write a changelog.

Is there a special PyPi only changelog separate from the general repo's changelog?

> If we did want to automate pushing to pypi, we'd have to put someone's credentials in there anyway, or create a separate bot account.

Having a `tilezen-bot` account is probably a generally good idea...

**Comment by nvkelso** [comment]
Related discussion in https://github.com/tilezen/tileserver/issues/115.

**Comment by rmarianski** [comment]
> Is there a special PyPi only changelog separate from the general repo's changelog?

No. I just meant that most of the work in making releases is in writing the changelogs.
